### ISSUE TITLE
Tearsheet: actions do not stick with scrolling content

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-cloud-cognitive#925]
## What package(s) are you using?

- [X] Cloud & Cognitive
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.
When the tearsheet has scrolling content, the ActionSet does not remain "sticky" or fixed to the bottom of the component. That is the expected behavior.
> Is this issue related to a specific component?
Tearsheet
> What did you expect to happen? What happened instead? What would you like to
> see changed?
I expected the ActionSet to remain fixed to the bottom of the tearsheet.
> What browser are you working in?
Chrome
> What version of the @carbon/ibm-cloud-cognitive packge are you using?
Latest

## Steps to reproduce the issue

1. View CreateTearsheet (with view all toggle on) to see an example OR have a Tearsheet component with scrolling content, and you'll notice the ActionSet does not remain visible at the bottom of the component (you have to scroll to see them, as opposed to them remaining fixed).

_Current_
![Screen Shot 2021-06-22 at 12 07 35 PM](https://user-images.githubusercontent.com/10215203/122961538-7e057d00-d352-11eb-9723-6da8c3bd40e9.png)
_Expected_
![Screen Shot 2021-06-22 at 12 07 27 PM](https://user-images.githubusercontent.com/10215203/122961576-82ca3100-d352-11eb-9708-9466e4b64e78.png)

**Comment by matthewgallo** [comment]
Another note, the influencer also scrolls when the main content area has scrolling content. I would expect the influencer section to remain fixed as well.

**Comment by dcwarwick** [comment]
Good catch @matthewgallo -- I think the restructuring I did to make a right-aligned influencer sit above the actions has lost some crucial scroll behaviour. I couldn't immediately fix the flex layout to restore this so I'm not sure whether it ever worked! Replacing the main content area with a grid layout sorts it though, and when the content scrolls correctly the influencer no longer scrolls off either.

I did notice two other glitches when checking out all the behaviours:
1. The influencers were ending up one pixel less than the designed widths, because of the 1px border. This meant that the side nav didn't quite fit and was overflowing by 1px horizontally. This wasn't a visible issue until I fixed the next point, when it became apparent. I've fixed the influencer widths in the tearsheet.
2. The side nav of the create tearsheet wasn't coping well with a short viewport -- instead of enabling a scroll, the toggle was absolutely positioned and was 'riding up' over the side nav which just got truncated. I applied the exact same fix as for the main content scrolling, making the influencer a grid layout and popping an extra div around the side nav to enable it to scroll neatly when the viewport is too short to show the whole side nav:
Before:
![image](https://user-images.githubusercontent.com/6385315/123009488-afaf3000-d3b4-11eb-8d0e-26ab7f9b9b9e.png)
After:
![image](https://user-images.githubusercontent.com/6385315/123009581-df5e3800-d3b4-11eb-854f-0deb1e40491b.png)
